packages/sso/src/saml.test.ts

```diff
@@ -610,7 +620,10 @@ describe("SAML SSO", async () => {
 		baseURL: "http://localhost:3000",
 		plugins: [bearer(), ssoClient()],
 		fetchOptions: {
-			customFetchImpl: async (url, init) => {
+			customFetchImpl: async (
```

Comment:
I think we don't need to declare types here? since `createAuthClient` should already do that